WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/LayoutPickerCategoryTableViewCell.swift

```diff
@@ -0,0 +1,125 @@
+    }
+
+    private func deselectItem(_ indexPath: IndexPath) {
+        collectionView.deselectItem(at: indexPath, animated: true)
+        collectionView(collectionView, didDeselectItemAt: indexPath)
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        if !selected, let selectedItems = collectionView.indexPathsForSelectedItems {
+            selectedItems.forEach { (indexPath) in
+                deselectItem(indexPath)
+            }
+        }
+
+        super.setSelected(selected, animated: animated)
+    }
+
+    func selectItemAt(_ position: Int) {
+        collectionView.selectItem(at: IndexPath(item: position, section: 0), animated: false, scrollPosition: [])
+    }
+}
+
+extension LayoutPickerCategoryTableViewCell: UICollectionViewDelegate {
+    func collectionView(_ collectionView: UICollectionView, shouldSelectItemAt indexPath: IndexPath) -> Bool {
+        if collectionView.cellForItem(at: indexPath)?.isSelected ?? false {
+            deselectItem(indexPath)
+            return false
+        }
+        return true
+    }
+
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        delegate?.didSelectLayoutAt(indexPath.item, forCell: self)
+    }
+
+    func collectionView(_ collectionView: UICollectionView, didDeselectItemAt indexPath: IndexPath) {
+        delegate?.didDeselectItem(forCell: self)
+    }
+}
+
+extension LayoutPickerCategoryTableViewCell: UICollectionViewDelegateFlowLayout {
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        return CGSize(width: 148.0, height: 230.0)
+     }
+}
+
+extension LayoutPickerCategoryTableViewCell: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return 30 // Static layouts currently only have one layout per category. Adding multiple in here to help test
+    }
+
+    func collectionView(_ LayoutPickerCategoryTableViewCell: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: LayoutPickerCollectionViewCell.cellReuseIdentifier, for: indexPath) as! LayoutPickerCollectionViewCell
+        let layout = layouts[0] // Static layouts currently only have one layout per category. Reusing the first to help test
+        cell.layout = layout
+        cell.isAccessibilityElement = true
+        cell.accessibilityLabel = layout.title + " \(indexPath.item)"
+        return cell
+    }
+}
+
+/// Accessibility
+extension LayoutPickerCategoryTableViewCell {
+    override func accessibilityElementDidBecomeFocused() {
+        delegate?.accessibilityElementDidBecomeFocused(forCell: self)
+    }
+}
+
+class AccessibleCollectionView: UICollectionView {
```

Comment:
This class is being used as a way to work around an issue where the collection view wasn't correctly registering the accessible elements when in a table view cell